gui/utils/compat.py

```diff
@@ -1,262 +0,0 @@
-# Backport of OrderedDict() class that runs on Python 2.4, 2.5, 2.6, 2.7 and pypy.
```

Comment:
There's a few bits of code that are geared for 2.6 and older (we actually try and import some 2.6 stuff and only except back to the 2.7 version...which is...weird....).  If you're okay with nuking it, I'm okay with nuking it. :)